src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -1136,6 +1161,15 @@ struct MHAHelper {
                         }
                     }
                     _gemv->tile_release();
+                } else if (_fastpath_valid_prec == ov::element::f16) {
```

Comment:
May we merge this branch to the above since the code is similar and no performance benefit?

Reply:
> May we merge this branch to the above since the code is similar and no performance benefit?

merge the similar codes.